test/test_pagination.js

```diff
@@ -59,6 +59,39 @@ testAsyncMulti('Pagination - previous/next controls', [function (test, expect) {
   Meteor.setTimeout(expectSecondPage, 0);
 }]);
 
+
+Tinytest.add('Pagination - rows per page', function (test) {
+  var table = Blaze.renderWithData(
+    Template.testTmpl,
```

Comment:
Why testTmpl instead of Template.reactiveTable? You can probably use my testTable helper if it doesn't need to be async.